test/e2e/kubectl/kubectl.go

```diff
@@ -1233,7 +1233,7 @@ metadata:
 			Testname: Kubectl, logs
 			Description: When a Pod is running then it MUST generate logs.
 			Starting a Pod should have a log line indicating the server is running and ready to accept connections. Also log command options MUST work as expected and described below.
-				‘kubectl log -tail=1’ should generate a output of one line, the last line in the log.
+				‘kubectl logs -tail=1’ should generate a output of one line, the last line in the log.
```

Comment:
this is still running `framework.RunKubectlOrDie("log"...` below.

@kubernetes/cncf-conformance-wg what's the protocol for removing a deprecated thing used by a conformance test? change the test in 1.15 and remove the thing in 1.16?

Reply:
No, we can just change the test now and remove the thing. Conformance tests evaluate whether the subject cluster conforms. If it can't handle the new command, it's not really conforming. If it can't handle the old one, and it is within the deprecation period (the CLI deprecation policy is the longer of 12 months or 2 releases (https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli)), then it wouldn't be considered conforming either. But 1) it's been more than 3 years so we are safely beyond that 12 months; and 2) this test isn't about backwards compat.

Reply:
I agree the command is within rights to be removed.

I just wasn't sure if 1.14 conformance tests needed to be able to run against 1.15 clusters.